domains/yt2mp3.is-an.app.json

```diff
@@ -9,6 +9,7 @@
   },
   "record": {
     "CNAME": "cname.vercel-dns.com."
```

Comment:
```suggestion
    "CNAME": "cname.vercel-dns.com.",
```

Reply:
I get this error:  ERROR: cannot have CNAME and TXT record with same name: yt2mp3.is-an.app
So, I need to remove the CNAME ?